content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+Have new Mozilla Firefox features 18 weeks before
+-------------------------------------------------
+
+Mozilla Firefox is on a fixed Release Calendar workflow called trains,
+which means that developper can plan their feature to land in a
+certain version and coordinate other teams work (QA, OPS, Service
+developers, Deployments, Translations) for a certain date.
+
+In case the deadline is missed, they will have to wait for the next
+release 6 weeks later.
+
+Then there are 4 versions of Firefox:
+
+- Firefox Nightly, which is a tag of Mozilla Central every day
+- Firefox Developer edition, which is a tag of Nightly every 6 weeks
+- Firefox Beta, which is a release of the previous Developer edition branch
+- Firefox Release, which is a release of the previous Beta branch
+
+With Nightly, the release is everyday, which means you will be the
+first one to benefit from the new feature, while other people will
+have to wait more than 3 months before being able to use it (less
+than 6 weeks in Nightly, 6 weeks for Developer edition, 6 weeks for
+Beta).
+
+Not to say that some features are only activated in Nightly.
+
+
+Make a difference and help reduce the number of crashes for Mozilla Firefox users
+---------------------------------------------------------------------------------
+
+The more users we have in Nightly, the more stable future versions of
+Firefox become.
+
+If everybody waits to use the last release of Firefox, it means
+that Developer Edition and Beta steps don't make any sense because
+they will not be used and bugs that might occur will appear directly
+at the Release stage, making a poor Firefox experience for people that
+don't even know what a browser is.
+
+By using Nightly, you will help us make a really great Firefox and as
+a benefit you will have this great Firefox experience 3 months ahead
+of schedule!
+
+Also most of the time, if you discover a problem with your Firefox
+Nightly, it will be fixed in the next release the day after, while
+with Firefox Release you will potentially have to wait for 6 weeks.
+
+Help add-ons developers to validate that their add-ons is still working
+-----------------------------------------------------------------------
+
+Firefox is transitionning from a complex and error prone way of
+writing add-ons to a standardized way of writing Web extensions, which
+are most likely to be compatible with Chrome and use well known Web
+technologies which makes them easier to develop.
+
+This is challenging for add-ons creators that already have a working
+add-on that needs to be rewritten.
+
+As a community we can help them. If they already have a Google Chrome
+add-on, it is most likely that they will be able to install it in
+Firefox directly. Plus, Firefox exposes more add-ons API
+than Chrome and we will create more in 2017 to help as much as
+possible port previous add-ons to the Web extensions standard.
+
+As a user we can help to test these new add-ons to keep a good
+experience for our users.
```

Comment:
You use `user` twice in this paragraph to mean two different groups of people. Maybe change the first for something like `early adopter`.